pkg/networkdevice/profile/profiledefinition/profile_definition.go

```diff
@@ -33,8 +33,7 @@ type ProfileDefinition struct {
 
 	// Version is the profile version.
 	// It is currently used only with downloaded/RC profiles.
-	// It's not exposed as yaml field since not necessary.
-	Version uint64 `yaml:"-" json:"version"`
+	Version uint64 `yaml:"version,omitempty" json:"version"`
```

Comment:
should the `json` field of this look like `json"version,omitempty"` following the pattern of the previous fields and `omitempty` being part of the `yaml` field?

Reply:
Since it wasn't `omitempty` previously I figured I'd leave it in for now; it doesn't really matter because at the moment we never jsonify these anyway.

Reply:
oooh okie makes sense, sounds good!